notebooks/official/migration/UJ4 Vertex SDK AutoML Tabular Binary Classification.ipynb

```diff
@@ -32,19 +32,64 @@
         "# Vertex AI: Vertex AI Migration: AutoML Tabular Binary Classification\n",
```

Comment:
<p>Combine these 2 cells.</p><p>Remove the 'Skip this step'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>